### PR TITLE
B2c return reason endpoint

### PR DESCRIPTION
**Pull request by himanshu-pal-metacube**
CC-16416 B2c return reason endpoint

### CODE REVIEW

File: resources/common/common_api.robot [hunk 1]
```diff
@@ -702,7 +702,7 @@ Each array element of array in response should contain property:
     ${list_length}=    Get Length    @{data}
     ${log_list}=    Log List    @{data}
     FOR    ${index}    IN RANGE    0    ${list_length}
-        ${list_element}=    Get From List    @{data}    ${index}
+        ${list_element}=    Get From List    @{data}    ${index}
```
Comment (OKrivtsova) [inline resources/common/common_api.robot:705]: please revert this change it brings o value only provokes merge conflicts

File: resources/environments/environments_api_b2c.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "environment": "api_b2c",
 
     "env": {
-        "glue_url": "http://glue.de.spryker.local",
+        "glue_url": "https://glue.de.b2c.internal-testing.demo-spryker.com",
```
Comment (OKrivtsova) [inline resources/environments/environments_api_b2c.json:5]: this should not be committed, please revert

File: resources/common/common_api.robot (hunk 1 above)
Comment (himanshu-pal-metacube) [reply resources/common/common_api.robot]: Done

File: resources/environments/environments_api_b2c.json (hunk 2 above)
Comment (himanshu-pal-metacube) [reply resources/environments/environments_api_b2c.json]: Done

**Review by OKrivtsova — APPROVED** [review]
